WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -197,47 +182,47 @@ private extension SupportTableViewController {
         }
 
         // Create and return table
-        let sections = [helpSection, forumsSection, informationSection, logOutSections].compactMap { $0 }
+        let sections = [forumsSection, informationSection, logOutSections].compactMap { $0 }
         return ImmuTable(sections: sections)
     }
 
     // TODO - remove after FeatureFlag.wordPressSupportForum is removed
```

Comment:
@guarani I think we can remove this TODO as well?

Reply:
Missed that one, thanks @sla8c! I pushed the change to remove this.